packages/modules/aave-v3/contracts/aave-v3-core/hardhat.config.cjs

```diff
@@ -1,5 +1,6 @@
 const { baseHardhatUserConfig } = require('@infinit-xyz/core/internal/hardhat-base-config')
 const { name } = require('../../package.json')
+require('@nomicfoundation/hardhat-verify')
```

Comment:
@Kittikornl Do we need to import hardhat-verify? I thought the config is only used when compile?